source/configuration_manual/replication.rst

```diff
@@ -0,0 +1,484 @@
+======================
+Replication with dsync
+======================
+
+.. note:
+
+  This is not supported with Dovecot Pro.
+
+Dovecot supports master/master replication using dsync. It's recommended
+that the same user always gets redirected to the same replica, but no
+changes get lost even if the same user modifies mails simultaneously on
+both replicas, some mails just might have to be redownloaded. The
+replication is done asynchronously, so high latency between the replicas
+isn't a problem. The replication is done by looking at Dovecot index
+files (not what exists in filesystem), so no mails get lost due to
+filesystem corruption or an accidental rm -rf, they will simply be
+replicated back.
+
+NOTE: v2.2 is highly recommended for this. Earlier versions can't do
+incremental metadata syncing. This means that the more mails a mailbox
+has, the slower it is to sync it.
```

Comment:
`.. note::` is used elsewhere. But should we just remove it already? Or at least change v2.2 to v2.2+?